examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,196 @@
+                cameraTransformOptions.coord.as(view.referenceCrs, cameraTransformOptions.coord);
+
+                if (cameraPositions.length > 0) {
+                    // Save the distance between the previous and the current picked points
+                    cameraTransformOptions.distance = view.camera.camera3D.position.distanceTo(
+                        cameraPositions[cameraPositions.length - 1],
+                    );
+                    cameraTransformOptions.easing = itowns.CameraUtils.Easing.Linear.None;
+                }
+
+                // Add the camera transform options and its position to the travel step arrays
+                cameraTravel.push(cameraTransformOptions);
+                cameraPositions.push(view.camera.camera3D.position.clone());
+
+                addCameraHelper()
+            }
+
+            function animateCamera() {
+                const camera = view.camera.camera3D;
+
+                // If travel is off, begin travel
+                if (!travelOn) {
+                    travelOn = true;
+                    // The travel time of each step is computed from the distance ran in the step and from the value of
+                    // cameraSpeed at the time of the travel.
+                    cameraTravel.forEach(function (item, index) {
+                        cameraTravel[index].time = (cameraTravel[index].distance || 0) / cameraSpeed * 1000;
+                    });
+
+                    // Return a promise for camera animation
+                    return itowns.CameraUtils.sequenceAnimationsToLookAtTarget(
+                        view,
+                        view.camera.camera3D,
+                        cameraTravel,
+                    ).then(() => {
+                        travelOn = false;
+                    });
+                }
+
+                // If travel is on, interrupt it and moves the camera to its initial placement :
+                itowns.CameraUtils.stop(view, view.camera.camera3D);
+                return itowns.CameraUtils.animateCameraToLookAtTarget(
+                    view,
+                    view.camera.camera3D,
+                    initialPlacement,
+                );
+            }
+
+            // Listens for user input :
+            window.addEventListener('keypress', async function (event) {
+                // If user presses `p` (for 'pick:'), saves the current camera position, heading, tilt...
+                if (event.key === 'p') { saveCurrentCameraTransformOptions() }
+                // If user presses `t` (for 'travel'), switches travelling mode on or off.
+                else if (event.key === 't') { animateCamera() }
+                // If user presses `c` (for 'clear'), removes all the picked position from lists.
+                else if (event.key ==='c') {
+                    if (!travelOn) {
+                        cameraTravel = [];
+                        cameraPositions = [];
+                        clearCameraHelper();
```

Comment:
use code method

Reply:
I removed the method and pasted its code here.